mailpoet/mailpoet
# Add tagging subscribers on the edit page [MAILPOET-4440]

This is the first pull request to add a new tag management feature.

**How to test**
1. Create/Edit a subscriber in the administration
2. Add/Remove tags
3. Check that tags are stored properly

Note: This PR contains a DB Migration. You need to run it before testing (e.g. deactivate and activate the plugin).

[MAILPOET-4440]

[MAILPOET-4440]: https://mailpoet.atlassian.net/browse/MAILPOET-4440?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
